pkg/cli/admin/release/git.go

```diff
@@ -163,8 +163,9 @@ func gitOutputToError(err error, out string) error {
 }
 
 var (
-	rePR     = regexp.MustCompile(`^Merge pull request #(\d+) from`)
-	rePrefix = regexp.MustCompile(`^(\[[\w\.\-]+\]\s*)+`)
+	squashRePR = regexp.MustCompile(`#(\d+)`)
```

Comment:
Looking at Insights history:

```console
$ git clone --depth 10 --branch release-4.11 https://github.com/openshift/insights-operator.git
$ cd insights-operator
$ git --no-pager log --no-merges --topo-order HEAD^^^..
commit 95961e7113ce179bf1621ac38842cad691d5b870 (HEAD -> release-4.11, origin/release-4.11)
Author: Tomas Remes <tremes@users.noreply.github.com>
Date:   Fri Apr 22 20:03:08 2022 +0200

    Fix vendoring of the build-machinery-go (#613)

commit 8f0ed50a6e1eaecf66650e364ee676339697c4ad
Author: Tomas Remes <tremes@users.noreply.github.com>
Date:   Thu Apr 21 16:23:10 2022 +0200

    Minor gatherer documentation update (#606)

commit 91b6fb4e66f52d44696ad6c3bd4cf06aae290dfe
Author: Tomas Remes <tremes@users.noreply.github.com>
Date:   Thu Apr 21 16:23:03 2022 +0200

    Create a new Prometheus metric providing Insights gathering time (#600)
    
    * Create a new Prometheus metric providing Insights gathering time
    
    * Rename the metric
```

Perhaps this regexp should require the wrapping parens too?

Reply:
This regexp extracts;
```
Fix vendoring of the build-machinery-go (#613)
Minor gatherer documentation update (#606)
Create a new Prometheus metric providing Insights gathering time (#600)
```

What do you mean by "wrapping parents"?

Reply:
It will also match 123 in

```
Fix vendoring from #123 (#613)
```

Using `[(]#(\d+)[)]` or similar will only match 613.

Reply:
You are totally right @wking, thanks. I'll open a fix PR.